src/components/Transaction/Transaction.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import { FiArrowUpRight } from 'react-icons/fi';
+import { MassaLogo } from '../Icons/Svg/Massa/MassaLogo';
+import { Transaction } from './Transaction';
+
+export default {
+  title: 'Components/Transaction',
+};
+
+const args = {
+  preIcon: <FiArrowUpRight />,
+  title: 'Transaction category',
+  address: '0x71c081f64f7bfe8ecb486029875f8e088db220649c62ab2c253da6c471ae137b',
+  posIcon: <MassaLogo size={24} />,
+  amount: '0,00',
+  timestamp: '2021-08-20T15:00:00.000Z',
+};
+
+export const _Primary = {
+  render: () => <Transaction {...args} customClass="!w-1/2" />,
```

Comment:
Is there any special reason force the override with important here!? **!w-1/2** 🤔

Reply:
It's just like "selector" component

Reply:
Oh I see! I think you can remove than. It will take the whole place on `width`

Reply:
okai, done